engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowInitializationActor.scala

```diff
@@ -54,10 +57,17 @@ case class WorkflowInitializationActor(workflowId: WorkflowId, workflowDescripto
   override val failureState = InitializationFailedState
   override val abortedState = InitializationsAbortedState
 
-  override val successResponse = WorkflowInitializationSucceededResponse
+  override def successResponse(data: WorkflowLifecycleActorData) = {
```

Comment:
Actually, I'd rather not.  😛 

After further discussion with @cjllanwarne and lingering remorse from so flagrantly having flouted the resolutions of our esteemed Google Doc, I'd like to have a TTT (Tuesday Tech Talk) to redo this a la Option 3, back-channel-service style.  The requirements of the service appear to be distinct from those in our existing service menagerie, so we'd either need to enhance an existing service or add a new one.